app/src/main/java/com/gcect/gcectapp/ui/fragments/HomeFragment.kt

```diff
@@ -218,6 +228,8 @@ class HomeFragment : Fragment(), OnHomeNoticeItemClickListener, OnBottomSheetIte
             .setOnProgressListener {
                 val percentage = it.currentBytes*100/it.totalBytes
                 pd.setMessage(getString(R.string.download_percentage_completed,percentage))
+                showNotification(percentage)
```

Comment:
Need to notify the update of progress , not needed to add notification everytime for the progress. This will resolve your issue that you are facing

Reply:
Ook thanks you I'll resolve all all these changes.

Reply:
> Need to notify the update of progress , not needed to add notification everytime for the progress. This will resolve your issue that you are facing

That's the problem how I'll notify can we have call or discord or meet?